src/borg/repository.py

```diff
@@ -1084,6 +1093,9 @@ class LoggedIO:
     class SegmentFull(Exception):
         """raised when a segment is full, before opening next"""
 
+    class AtticSegment(Error):
+        """Attic segment detected [segment {}]"""
```

Comment:
the idea was just that it looks more consistent with the other magic related exception message.